src/catalog/src/lib.rs

```diff
@@ -254,6 +257,7 @@ pub async fn datanode_stat(catalog_manager: &CatalogManagerRef) -> (u64, Vec<Reg
                                 table_name: table_name.clone(),
                             }),
                             approximate_bytes: stat.disk_usage_bytes as i64,
+                            attrs: HashMap::from([(stat.region_id.to_string(), engine.clone())]),
```

Comment:
Since the `attrs` is an internal property of the region, perhaps 'engine_name' should be used as the key?

Reply:
Thanks for your help, I'll rewrite it.